fastlane/Fastfile

```diff
@@ -228,10 +228,10 @@ desc "Verifies all tests pass and the current state of the repo is valid"
 private_lane :validate_repo do
   # Verifying that no debug code is in the code base
   #
-  ensure_no_debug_code(text: "binding.pry", extension: ".rb", exclude: "**/*/playground.rb") # debugging code
-  ensure_no_debug_code(text: "# TODO", extension: ".rb") # TODOs
-  ensure_no_debug_code(text: "now: ", extension: ".rb") # rspec focus
-  ensure_no_debug_code(text: "<<<<<<<", extension: ".rb") # Merge conflict
+  ensure_no_debug_code(text: "binding.pry", extension: ".rb", exclude: "(.bundle/**/*.rb|**/*/playground.rb)") # debugging code
+  ensure_no_debug_code(text: "# TODO", extension: ".rb", exclude: "\*/.bundle*") # TODOs
```

Comment:
Should we install the bundler in a root directory, so that it's not included by default when searching through files?